packages/cli/create-strapi-app/src/create-strapi-app.ts

```diff
@@ -106,5 +107,6 @@ async function initProject(projectName: string, programArgs: Program) {
     ...options,
   };
 
-  return generateApp(directory, generateStrapiAppOptions);
+  await handleCloudProject(directory);
```

Comment:
just wondering why we decided to add it here instead of in the generateApp code ?

Reply:
That's a good idea